static/js/util/util.js

```diff
@@ -600,3 +600,18 @@ export const mapObj = R.curry((fn, obj) =>
  */
 export const wait = (millis: number): Promise<void> =>
   new Promise(resolve => setTimeout(resolve, millis))
+
+/**
+ * extract object from json
+ */
+export const findObjByName = (list: Object, key: string) => {
+  if (_.has(list, key)) {
+    return [list]
+  }
+  return _.flatten(
+    _.map(list, function(value) {
+      return typeof value === "object" ? findObjByName(value, key) : []
+    }),
+    true
```

Comment:
I think here instead of `_flatten(_map` you might be able to get away with just using `_.flatMap`. What's the second `true` argument to `_.flatten` for? in the docs it looks like it only takes a single argument (the list to flatten).

Reply:
I might also suggest renaming the `list` parameter, since it isn't always a list